Client/Assets/CSS/loggedin-base.scss

```diff
@@ -41,4 +42,22 @@
             text-decoration: none;
         }
     }
+
+    @media (min-width: 992px) {
```

Comment:
@xh3n1  can this be moved into a variable? :)

Reply:
@LukasReschke sure - but I will address the "breakpoints" variables in a follow up PR ;)